<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>svmInfo</class>
 <widget class="QWidget" name="svmInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>201</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>VanshingPoints</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="x1_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vp_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="x3_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="x4_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X4</string>
    </property>
   </widget>
   <widget class="QLabel" name="x2_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="x4_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="x1_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="x2_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>X2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="x3_check">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>48</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>X3</string>
    </property>
   </widget>
   <widget class="QLabel" name="vpx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>35</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>VPX</string>
    </property>
   </widget>
   <widget class="QComboBox" name="vp">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>VanishingPointX</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VanishingPointY</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>VanishingPointZ</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
